rust/src/lib/nm/error.rs

```diff
@@ -37,6 +37,22 @@ pub(crate) fn nm_error_to_nmstate(nm_error: NmError) -> NmstateError {
                 ),
             )
         }
+        NmErrorKind::Connection(NmConnectionError::InvalidProperty) => {
+            if nm_error.msg.starts_with("ipv4.forwarding") {
+                NmstateError::new(
+                    ErrorKind::NotSupportedError,
+                    format!(
+                        "Please upgrade NetworkManager to 1.54 or above to \
+                        support IPv4 sysctl forwarding as indicated by: {nm_error}",
+                    ),
+                )
```

Comment:
This is not very reliable. It is good to explain that the problem is the NM version. However, this message will be printed for all errors related to ipv4.forwarding, so if it's a different cause it will confuse the user, instead of helping him.
There is no good way to properly detect this, other than [this](https://github.com/nmstate/nmstate/blob/2d8e556f931baf10f20db1aee9fd48c5fb5724c3/rust/src/lib/nm/query_apply/apply.rs#L491), but it's a bit overkill in this case. I would just show the error message returned by NM, as it is done by default, and that's all.

Reply:
I agree on showing NM error message unmodified.

Reply:
If we show the unmodified message from NM, there is no need to check if the error is for ipv4.forwarding or not. The only difference is that you are using `ErrorKind::NotSupportedError` but, again, we don't know if that's the actual cause or not. We only know that the error is related to forwarding, nothing else.
Better just remove this change.